### PR TITLE
Documentation update for 0.1.2 release

### PR DESCRIPTION
**Pull request by chinmayshah99**
# Pull Request

## Description
Please include a summary of the change and which issue is fixed. If this closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## Type of Change
Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation (non-breaking change which adds documentation)
- [ ] Improvement (non-breaking change that improves the performance or reliability of existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I did follow the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md)
- [ ] I did follow the [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have added tests for my changes

## Additional Context
Please also include relevant motivation and context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In today's data-driven world, where data analytics is used by the Researchers or
 This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, helping us achieve better privacy.
 
 **Things to remember about PyDP :**
-- :rocket: Features Differentially Private Algorithmic functions as part of the 0.1.1 dev release, added to carrots demo such as BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median , etc.  
+- :rocket: Features Differentially Private Algorithmic functions to support carrots demo including: BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median, etc.  
 - All the computation methods mentioned above, use Laplace noise only. (Other noise mechanisms will be added soon... :smiley:)
```
Comment (jeamick) [inline README.md:14]: Let's add a link for the others mechanism or paper so people have an idea about what does it means:
- https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf
- https://en.wikipedia.org/wiki/Additive_noise_mechanisms

**Review by jeamick — CHANGES_REQUESTED** [review]
Looks good to me! I just added a little comment, let me guys know your thoughts about it.

File: README.md (hunk 1 above)
Comment (shaistha24) [reply README.md:14]: Great, I think we can add it to the learning resources @jeamick  - to know more about DP and stuff. The readme should only describe the project. What do you think @chinmayshah99  ?

I think the current description is enough. But, as always any suggestions are welcome!

Comment (chinmayshah99) [reply README.md:14]: Yeah @shaistha24 , we can add this to learning resources, and create a link from here in readme. 
@shaistha24 Can you help us out here?

**Comment by chinmayshah99** [comment]
While I have merged this PR, @shaistha24 can you add links to explore further section and link it with the main readme?

**Pull request merged**